gcore/src/prog.rs

```diff
@@ -107,7 +110,7 @@ pub fn create_program_with_gas(
     payload: &[u8],
     gas_limit: u64,
     value: u128,
-) -> ActorId {
+) -> Result<ActorId, ExtError> {
```

Comment:
I would provide custom `Result<T>` for `gcore` also, replacing other explicit Result<T, ExtError> afterwards